jobs-staging.yml

```diff
@@ -16,6 +16,8 @@ jobs:
 
 
 properties:
+  collectd:
+    riemann_server: 0.monitoring.monitoring.monitoring-staging.bosh
```

Comment:
I made this value up...seems like a staging Riemann instance might not exist yet? Found `monitoring-prod` in [that manifest](https://github.com/18F/cg-deploy-monitoring/blob/49bd8cbf5af110e4ddd1bacea443f5364ee3b0e5/monitoring.yml#L1), but not `monitoring-stage` or `monitoring-staging`. Potentially due to https://github.com/18F/cg-deploy-monitoring/issues/7?

/cc @sharms